NewProfile/index.ts

```diff
@@ -0,0 +1,48 @@
+import * as crypto from "crypto";
+import { AzureFunction, Context } from "@azure/functions";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+
+const Failure = t.interface({
+  kind: t.keyof({
+    PERMANENT: null,
+    TRANSIENT: null
+  }),
+  reason: t.string
+});
+type Failure = t.TypeOf<typeof Failure>;
+
+const Success = t.interface({
+  kind: t.literal("SUCCESS")
+});
+type Success = t.TypeOf<typeof Success>;
+
+const permanentDecodeFailure = (errs: t.Errors): Failure =>
+  Failure.encode({
+    kind: "PERMANENT",
+    reason: `Cannot decode input: ${readableReport(errs)}`
+  });
+
+const index: AzureFunction = async (
+  context: Context,
+  inputFiscalCode: unknown
```

Comment:
To allow future improvement, I'd rather use an object with a parameter:
```
{ fiscalCode: "dsdsd" }
```

Reply:
I like it, but we need to share this type across 2 functions (`fn-covidcert` and `fn-app`). Can we define this type inside `fn-commons`?

Reply:
Fixed in 7b3021a